### PR TITLE
Increased coverage for app component from 31% to 77%

### LINKED ISSUE #9: Add test cases - App component
Need to add test cases for all components

### PR DESCRIPTION
**Pull request by AlexandreInsua**
Hi, I was working in unit tests.
I have left two commented tests in case you want to do the refactorings that would be necessary for the app to pass them.
I would like to continue working on this project.
Could I keep writing tests?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,4 +53,4 @@
     "tslint": "~5.11.0",
     "typescript": "~3.1.1"
   }
-}
\ No newline at end of file
+}
```
Comment (shubhamjsx) [inline package.json]: @AlexandreInsua Can you please remove package.json and package-lock.json from the commit?

**Review by shubhamjsx — CHANGES_REQUESTED** [review]
Sure, please continue. I have starting writing test for item-edit component, please take it up after I commit it. For other components please feel free to continue.

**Comment by AlexandreInsua** [comment]
Done.

**Comment by AlexandreInsua** [comment]
Hi, @shubham-kumar10 .  I go to write tests for signup component.

**Pull request merged**